packages/pebble-shared/svgs/schedule-mail.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1000pt" height="1000pt" viewBox="0 0 1000 1000"><path d="M570.387 819.055c-22.574 8.015-46.262 12.082-70.387 12.082-116.352 0-210.96-95.012-210.96-211.766 0-116.75 94.608-211.766 210.96-211.766s210.957 95.016 210.957 211.766c0 51.883-42.035 94.117-93.77 94.117-22.156 0-42.539-7.765-58.593-20.699-16.055 12.934-36.434 20.7-58.594 20.7-51.734 0-93.773-42.235-93.773-94.118 0-51.879 42.039-94.117 93.773-94.117h58.594c19.449 0 35.176 15.824 35.176 35.293v58.824c0 12.988 10.535 23.527 23.418 23.527 12.882 0 23.414-10.539 23.414-23.527 0-77.863-63.102-141.176-140.602-141.176s-140.602 63.313-140.602 141.176c0 77.86 63.102 141.176 140.602 141.176 16.121 0 31.895-2.707 46.902-8.035 18.328-6.52 38.438 3.136 44.91 21.508 6.47 18.351-3.101 38.523-21.425 45.035zm0 0h-.004l-1.969-5.547zM523.414 619.37c0 12.988-10.531 23.527-23.414 23.527-12.883 0-23.414-10.539-23.414-23.527 0-12.988 10.531-23.531 23.414-23.531h23.414zm447.293-505.883h-83.965v-23.53c0-19.47-15.73-35.294-35.176-35.294-19.453 0-35.183 15.824-35.183 35.293v23.531H710.957v-23.53c0-19.47-15.727-35.294-35.176-35.294-19.453 0-35.18 15.824-35.18 35.293v23.531H535.18v-23.53c0-19.47-15.73-35.294-35.18-35.294s-35.18 15.824-35.18 35.293v23.531H359.398v-23.53c0-19.47-15.73-35.294-35.18-35.294-19.448 0-35.179 15.824-35.179 35.293v23.531H183.617v-23.53c0-19.47-15.73-35.294-35.18-35.294-19.449 0-35.18 15.824-35.18 35.293v23.531h-83.96c-19.45 0-35.18 15.825-35.18 35.293v764.707c0 19.47 15.73 35.293 35.18 35.293h941.41c19.445 0 35.176-15.824 35.176-35.293V148.781c0-19.468-15.73-35.293-35.176-35.293zM64.477 878.195V360.547h871.046v517.648zm871.046-588.238H64.477V184.078h48.78v23.527c0 19.47 15.731 35.293 35.18 35.293 19.45 0 35.18-15.824 35.18-35.293v-23.527H289.04v23.527c0 19.47 15.73 35.293 35.18 35.293 19.449 0 35.18-15.824 35.18-35.293v-23.527H464.82v23.527c0 19.47 15.73 35.293 35.18 35.293s35.18-15.824 35.18-35.293v-23.527h105.422v23.527c0 19.47 15.726 35.293 35.18 35.293 19.448 0 35.175-15.824 35.175-35.293v-23.527h105.426v23.527c0 19.47 15.73 35.293 35.183 35.293 19.446 0 35.176-15.824 35.176-35.293v-23.527h48.781zm0 0" fill="#000" stroke-width="11.764706" stroke="#000"/></svg>
```

Comment:
`@` symbol not drawn properly

Reply:
for now adding it. DIscussed it with Mandeep we can modify it later.